packages/trader/src/Modules/SmartChartBeta/Components/LaunchModal/__tests__/launch-modal.spec.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import LaunchModal from '../launch-modal';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TraderProviders from '../../../../../trader-providers';
+
+const renderLaunchModal = (is_logged_in = true) => {
+    const mocked_store = mockStore({
+        client: { is_logged_in },
+    });
+
+    const handleMockLaunchModal = () => {
+        sessionStorage.setItem('mockLaunch', JSON.stringify(true));
+    };
+
+    render(
+        <StoreProvider store={mocked_store}>
+            <LaunchModal open={true} handleChange={handleMockLaunchModal} />
+        </StoreProvider>
+    );
+};
+
+const renderTradeComponent = (storeProps = {}) => {
+    const mocked_store = mockStore({});
+    render(
+        <StoreProvider store={mocked_store}>
+            <TraderProviders store={mockStore(storeProps)} />
+        </StoreProvider>
+    );
+};
+
+jest.mock('Assets/SvgComponents/launch/ic-chart-launch.svg', () => {
+    const LaunchModalChartImage = () => <div>Chart Svg</div>;
+
+    return LaunchModalChartImage;
+});
+
+describe('Launch Modal', () => {
+    let modal_root_el: HTMLDivElement;
```

Comment:
if `modal_root_el` is used only inside of `beforeAll()` what is the purpose of declaring it outside?